common/app/model/meta.scala

```diff
@@ -59,12 +59,29 @@ object Fields {
       displayHint = apiContent.fields.flatMap(_.displayHint).getOrElse(""),
       isLive = apiContent.fields.flatMap(_.liveBloggingNow).getOrElse(false),
       sensitive = apiContent.fields.flatMap(_.sensitive),
-      shouldHideReaderRevenue = apiContent.fields.flatMap(_.shouldHideReaderRevenue),
+      shouldHideReaderRevenue = shouldHideReaderRevenue(apiContent),
       legallySensitive = apiContent.fields.flatMap(_.legallySensitive),
       firstPublicationDate = apiContent.fields.flatMap(_.firstPublicationDate).map(_.toJodaDateTime),
       lang = apiContent.fields.flatMap(_.lang)
     )
   }
+
+  private def shouldHideReaderRevenue(apiContent: contentapi.Content): Option[Boolean] = {
```

Comment:
just curious here. It looks like you are never returning `None`, so I was wondering if just returning a `Boolean` would be fine?